tendrl/gluster_integration/objects/gluster_brick/atoms/create/__init__.py

```diff
@@ -87,8 +88,13 @@ def run(self):
                         pool=val["pool"],
                         pv=val["pv"],
                         **args
-                    ).save(update=False)
-
+                    ).save()
+                    free_brick_key = "clusters/%s/nodes/%s/GlusterBricks/free/%s" % (
+                        NS.tendrl_context.integration_id,
+                        val["node_id"],
+                        brick_name.replace("/","_")
+                    )
+                    NS._int.wclient.write(free_brick_key, "")
```

Comment:
We have following things under gluster_bricks:
- free: containg just the name of free bricks
- used: containg just the name of used bricks
- all: contains all the details of all the bricks
free and used are kind of index.
Here when I create brick I update the detail under "all" and free section